vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -4,30 +4,36 @@ sidebarDepth: 2
 # Export and Publish a Bundle
 
 ## Overview
-Use the Entando bundler command to export a bundle of Entando components from an existing Entando application. An Entando bundle can be used to do the initial install of Entando components into an Entando application, migrate Entando components from one environment to another (e.g. Dev to QA), to provide a template for building a new Entando application, or as the skeleton of an Entando solution. The output of this command is the same bundle directory structure created by an Entando project including a bundle descriptor file.
+Use the bundler command to export a bundle of Entando components from an existing Entando Application. An Entando Bundle can be used 
+- For the initial install of Entando components into an Entando Application
+- To migrate Entando components from one environment to another (e.g. Dev to QA)
+- To provide a template for building a new Entando Application
+- As the skeleton of an Entando solution
+
+This command generates the same bundle directory structure as an Entando project, including a bundle descriptor file.
 
 ### Prerequisites
-* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify you have the prerequisites in place for this tutorial (e.g. git, entando-bundler). 
+* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify prerequisites (e.g. git, entando-bundler): 
 ``` sh
 ent check-env develop 
 ```
-* You'll also need a running Entando application.
-* You'll need admin access to Keycloak or admin access to Kubernetes to set it up.
+* A running Entando Application
+* Admin access to Keycloak or admin access to Kubernetes to set it up
```

Comment:
>> Admin access to Keycloak or Kubernetes for setup

Reply:
i think we'd have to rephrase if you want to drop the second "admin access"; "for setup" doesn't let you know you'll be setting up admin access to Keycloak via Kubernetes

"admin access to Keycloak via credentials or through Kubernetes"? not sure that's an improvement. @nshaw

Reply:
I think we can simplify to "Admin access to Keycloak". We could link to https://developer.entando.com/v7.0/docs/consume/identity-management.html#logging-into-your-keycloak-instance but I don't know if that's overkill.